support-frontend/assets/pages/paper-subscription-landing/components/content/collectionTab.jsx

```diff
@@ -7,10 +7,10 @@ import React from 'react';
 import Content from 'components/content/content';
 import Text from 'components/text/text';
 import GridImage from 'components/gridImage/gridImage';
-import { sendClickedEvent } from 'helpers/tracking/clickTracking';
+// import { sendClickedEvent } from 'helpers/tracking/clickTracking';
```

Comment:
Could this be removed now?

Reply:
Or does the tracking still need to be implemented? If so, probably need to do that before merging in.

Reply:
As far as I can see this is the only place we're currently recording tracking events for clicking on FAQs or contact links- I'll check with Matt if we want to keep this or get rid.

Reply:
Matt says there's no need to keep it so I'm getting rid, I also realised that the same FAQs/Terms & Conditions component was replicated on the Home Delivery and Subs Card tabs so those have both been sorted out now too.

Reply:
👍